packages/protocol/contracts/L1/TaikoL1.sol

```diff
@@ -38,6 +38,9 @@ contract TaikoL1 is EssentialContract, IHeaderSync, V1Events {
     ) external initializer {
         EssentialContract._init(_addressManager);
         V1Finalizing.init(state, _genesisBlockHash);
+
+        state.whitelistProposers = false;
```

Comment:
since all of this will be removed, maybe it's nicer to just make `V1Whitelisting.sol` and put everything in there and link back to that from TaikoL1, and then we can just remove "V1Whitelisting" when public testnet is done, so we don't miss anything? But we can also jsut revert this PR if ther arent too many change and it can safely revert.

Reply:
Even if that's the case, we still have to change multiple files including TaikoL1.sol, and the deployment script.